presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -71,22 +98,38 @@ Array Functions
 
     Returns the cardinality (size) of the array ``x``.
 
+        SELECT cardinality(ARRAY[10,20,30,40,50],10); -- 5
+        SELECT cardinality(ARRAY['e',null,'c','b','a']); -- 5
+
 .. function:: concat(array1, array2, ..., arrayN) -> array
     :noindex:
 
     Concatenates the arrays ``array1``, ``array2``, ``...``, ``arrayN``.
     This function provides the same functionality as the SQL-standard concatenation operator (``||``).
 
+        SELECT concat(ARRAY[1,1,2], ARRAY[2,3]); --  [1, 1, 2, 2, 3]
+        SELECT concat(ARRAY[1,1,2], ARRAY[2,null]); -- [1, 1, 2, 2, null]
+        SELECT concat(ARRAY['a','a','b'], ARRAY['b', 'c']); -- ['a', 'a', 'b', 'b', 'c']
+
 .. function:: contains(x, element) -> boolean
 
     Returns true if the array ``x`` contains the ``element``.
 
+        SELECT contains(ARRAY[1,1,2], 3); -- false
+        SELECT contains(ARRAY['a',null,'b'], 'a'); -- true
+        SELECT contains(ARRAY['a',null,'b'], null); -- NULL
+
 .. function:: element_at(array<E>, index) -> E
 
     Returns element of ``array`` at given ``index``.
     If ``index`` >= 0, this function provides the same functionality as the SQL-standard subscript operator (``[]``).
     If ``index`` < 0, ``element_at`` accesses elements from the last to the first.
 
+        SELECT element_at(ARRAY[10,20,30,40,50],1); -- 10
+        SELECT element_at(ARRAY[10,20,30,40,50],-3); -- 30
+        SELECT element_at(ARRAY[10,20,30,40,50],10); -- NULL
```

Comment:
I personally prefer to make the index just beyond the array size, e.g. `6` and `-6` :)